wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1382,41 +1383,51 @@ def keyboard_shortcuts_dialog(context):
 
     user_agent = context["request"].headers.get("User-Agent", "")
     is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
-    modifier = "⌘" if is_mac else "Ctrl"
+    KEYS = get_keyboard_key_labels_from_request(context["request"])
+
+    comments_enabled = get_comments_enabled()
 
     return {
         "shortcuts": {
             ("actions-common", _("Common actions")): [
-                (_("Copy"), f"{modifier} + c"),
-                (_("Cut"), f"{modifier} + x"),
-                (_("Paste"), f"{modifier} + v"),
+                (_("Copy"), f"{KEYS.MOD} + c"),
+                (_("Cut"), f"{KEYS.MOD} + x"),
+                (_("Paste"), f"{KEYS.MOD} + v"),
                 (
                     _("Paste and match style")
                     if is_mac
                     else _("Paste without formatting"),
-                    f"{modifier} + Shift + v",
+                    f"{KEYS.MOD} + {KEYS.SHIFT} + v",
                 ),
-                (_("Undo"), f"{modifier} + z"),
+                (_("Undo"), f"{KEYS.MOD} + z"),
                 (
                     _("Redo"),
-                    f"{modifier} + Shift + z" if is_mac else f"{modifier} + y",
+                    f"{KEYS.MOD} + {KEYS.SHIFT} + z" if is_mac else f"{KEYS.MOD} + y",
                 ),
             ],
             ("actions-model", _("Actions")): [
-                (_("Save changes"), f"{modifier} + s"),
-                (_("Preview"), f"{modifier} + p"),
+                (_("Save changes"), f"{KEYS.MOD} + s"),
+                (_("Preview"), f"{KEYS.MOD} + p"),
+                (
+                    _("Comments"),
+                    f"{KEYS.MOD} + {KEYS.ALT} + m",
```

Comment:
```suggestion
                    f"{KEYS.CTRL} + {KEYS.ALT} + m",
```

Sorry, this probably got lost in the refactor, it should be ctrl + alt (or ctrl + opt on mac).

https://github.com/wagtail/wagtail/blob/5c30b4211a36cab32727e3a155f70464295e446a/client/src/entrypoints/admin/comments.js#L16-L18